drivers/gles3/storage/config.cpp

```diff
@@ -99,8 +99,14 @@ Config::Config() {
 	msaa_supported = extensions.has("GL_EXT_framebuffer_multisample");
 #endif
 #ifndef IOS_ENABLED
+#ifdef WEB_ENABLED
+	msaa_multiview_supported = extensions.has("OCULUS_multiview");
+	rt_msaa_multiview_supported = msaa_multiview_supported;
```

Comment:
Is this correct David? `rt_msaa_multiview_supported` uses an approach that may not be supported by all drivers, or does webgl require support for these?

Reply:
Ah, cause it's part of the OCULUS_multiview spec. Just saw the code further down